src/index.ts

```diff
@@ -251,7 +253,7 @@ function checkNodeVersion(): void {
         return;
     }
 
-    if (!nodeVersionUtils.majorVersionEqual()) {
+    if (!nodeVersionUtils.majorVersionEqual(cplaceVersion)) {
```

Comment:
If cplace version is e.g. 5.10 the program will continue checking [minor](https://github.com/collaborationFactory/cplace-asc/blob/1540e54472d254c127eaa37e8ef3405985a4b003/src/index.ts#L263) and [patch](https://github.com/collaborationFactory/cplace-asc/blob/1540e54472d254c127eaa37e8ef3405985a4b003/src/index.ts#L269) version and the warning will be logged. We should prevent those warnings for versions <= 5.12